examples/bsttest.wyv

```diff
@@ -0,0 +1,31 @@
+require stdout
```

Comment:
please put in an appropriate subdirectory of examples.  Maybe introductory, or a new "data-structures" directory?